tests/app/test_app.py

```diff
@@ -464,7 +449,7 @@ def test_if_user_unknown_in_party_service_no_email_sent(self, mock_alerter, mock
                                                                        "lastName": "",
                                                                        "telephone": "+443069990888",
                                                                        "status": "ACTIVE",
-                                                                       "sampleUnitType": "BI"}, 200))
+                                                                       "sampleUnitType": "BI"}))
```

Comment:
Why is sample unit type necessary, as secure message will never deal with the B case only BI?